spec/models/asp/payment_request_state_machine_spec.rb

```diff
@@ -14,7 +14,9 @@
       let(:asp_payment_request) { create(:asp_payment_request, :incomplete) }
 
       it "allows the transition" do
-        expect { asp_payment_request.mark_ready! }.not_to raise_error Statesman::TransitionFailedError
```

Comment:
alors c'était subtil ce test c'était pour dire que ça plante mais pas avec cette erreur (it allows the transition, but it rases the GuardFailure, test d'après) c'est hyper confus je m'en rends compte maintenant

Reply:
du coup laisse comme t'as fait c'est bien

Reply:
Trop subtil ! Même pour RSpec qui m'a donné le tuyau